src/boot/efi/util.c

```diff
@@ -411,8 +411,16 @@ EFI_STATUS file_read(EFI_FILE_HANDLE dir, const CHAR16 *name, UINTN off, UINTN s
         return err;
 }
 
+VOID PrintError(const CHAR16 *format, ...) {
```

Comment:
Maybe call this also add "Stall" to the name. It's a relevant fact, and a bit unexpected in particular when we compare this with log_error() in the rest of our codebase. Hence PrintErrorStall()?